zero_bin/ops/src/lib.rs

```diff
@@ -136,35 +135,24 @@ impl SegmentProofSpan {
     ///
     /// Either the first 8 characters of the hex-encoded hash of the first and
     /// last transactions, or "Dummy" if there is no transaction.
-    fn get_descriptor(ir: &GenerationInputs) -> String {
-        if ir.signed_txns.is_empty() {
+    fn get_descriptor(ir: &TrimmedGenerationInputs) -> String {
+        if ir.txns_len == 0 {
             "Dummy".to_string()
-        } else if ir.signed_txns.len() == 1 {
+        } else if ir.txns_len == 1 {
+            format!("{:x?}", ir.txn_number_before)
+        } else {
             format!(
-                "{:x?}",
-                u64::from_be_bytes(keccak(&ir.signed_txns[0][..])[0..8].try_into().unwrap())
+                "{:?}..{:?}",
+                ir.txn_number_before,
+                ir.txn_number_before + ir.txns_len
```

Comment:
This is actually redundant information, as we already have access to the indices in the `id` printed along with the descriptor. What we could do instead is compute the hashes on the `evm_arithmetization` side and plug them into `TrimmedGenerationInputs`. That would also remove the `txns_len` field that could be inferred directly.

Reply:
Addressed in latest commit.

Reply:
I'm not sure I understand what you mean here: Do you want to compute the hash of all transactions (which I believe are only used for logging information) on evm_arithmetization side and pass them to the TrimmedGenerationInputs instead of just the length?

Reply:
Yeah as the computation is being done anyway, this is not an overhead and allows to still refer to the hashes with `TrimmedGenerationInputs`.

Reply:
This is technically cheaper as we just clone a few H256 instead of hashing all the time